### PR TITLE
Lotfp saves fix

### PR DESCRIPTION
**Pull request by Haplo-Dragon**
LotFP characters have save values that occasionally differ from those in original D&D. I've added the correct LotFP saves in characterclass.py, and fixed character.py so that the LotFP get_saves method grabs the LotFP-specific values.

I also noticed a bug in the get_saves method for LotFP characters - the int_bonus and wis_bonus variables were both referencing WIS. I fixed it so that int_bonus references INT.

This is my first-ever commit and pull request, so please let me know if I've done something wrong, or need to add or remove something.

Thank you so much for creating an amazing tool that I use all the time! My group loves it, and so do I. 

Ethan
Texas, USA

### CODE REVIEW

**Comment by Haplo-Dragon** [comment]
Oops! I noticed a few more minor things. 
LotFP characters get their INT bonus added to their Languages skill, so I've added that, and Magic Missile was misspelled in the LotFP spell list.
Elf characters start with only Read Magic, so I've changed the get_spells method in character.py, too.

I also realized that the results of the latest merge don't seem to be taking effect at http://character.totalpartykill.ca/lotfp. That is, the Architecture spelling fix for Dwarf characters isn't reflected over there, even though it is here in the repo. I figured maybe you hadn't pushed that update to your server yet.

Ethan

**Comment by funkaoshi** [comment]
I should check github more often!

File: character.py [hunk 1]
```diff
@@ -286,9 +290,9 @@ def get_saves(self):
         Your magic based saves are effected by your INT, other saves by your
         WIS.
         """
-        saves = copy.copy(self.character_class['saves'])
+        saves = copy.copy(characterclass.LOTFP['lotfp_saves'][self.character_class['name']])
         wis_bonus = self.get_bonus(*self.attributes[characterclass.WIS])
-        int_bonus = self.get_bonus(*self.attributes[characterclass.WIS])
+        int_bonus = self.get_bonus(*self.attributes[characterclass.INT])
```
Comment (funkaoshi) [inline character.py:295]: Doh.

**Comment by funkaoshi** [comment]
Thanks again!

**Comment by Haplo-Dragon** [comment]
Any time!

**Pull request merged**